static/app/views/dashboards/manage/index.tsx

```diff
@@ -63,18 +66,74 @@ type State = {
   dashboards: DashboardListItem[] | null;
   dashboardsPageLinks: string;
   showTemplates: boolean;
+  windowWidth: number;
 } & DeprecatedAsyncView['state'];
 
 class ManageDashboards extends DeprecatedAsyncView<Props, State> {
+  shouldReload = true;
+
+  constructor(props: Props, context: any) {
+    super(props, context);
+
+    this.state = {
+      ...super.getDefaultState(),
+      ...this.getDefaultState(),
```

Comment:
It looks like `this.getDefaultState` already calls super, so we could probably drop the super call here.